scripts/terraform_plan_comment_update.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+
+MESSAGE=`terraform plan -chdir="$1" -no-color | ./scripts/redact-output.sh | grep '^Plan:'`
```

Comment:
Is it worth using the output of `terraform-plan.sh` here, as it already has the correct flags set and redacts the output? It means we'd only need to update `terraform-plan.sh` rather than this file too for the same output?

Reply:
updated to reference scripts/terraform_plan.sh